packages/tools/devtools/devtools-core/src/data-visualization/VisualTree.ts

```diff
@@ -196,7 +196,7 @@ export interface FluidUnknownObjectNode extends FluidObjectNodeBase {
 /**
  * Node pointing to another Fluid object via a unique identifier.
  *
- * @public
+ * @internal
```

Comment:
i picked this one at random, but its not used outside of the package, can we remove most of these from the exported surface, as marking internal currently does nothing.

Reply:
things like this will actually make it hard to make internal do something, as turning it on will be a huge breaking change.

Reply:
These types are used by the extension. They aren't intended to be used by external consumers of the package, but they are used in the messages passed between `devtools-core` and `devtools-view`.

Reply:
ahh. i see it now, in the tsx file. i had . *.ts filter :(

Reply:
I suspect an audit of the exports here is due, but many of them are types to ensure the messaging between the two packages is type-safe 😕, so I think most of what is exported is needed.